io.github.ryanabx.flatrun.yml

```diff
@@ -0,0 +1,79 @@
+id: io.github.ryanabx.flatrun
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: org.flatpak.Builder.BaseApp
+base-version: "23.08"
+separate-locales: false
+command: flatrun
+finish-args:
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+# Unfortunately, it's necessary for us to have this permission
+# all unsandboxed access is run through flatrun-agent.
+# flatrun-agent installs system runtimes and launches flatpaks
+# from a local repo (non system and non user).
+  - "--talk-name=org.freedesktop.Flatpak"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/flatrun/cargo"
+modules:
+# Install flatpak
+  - name: flatpak
+    config-opts:
+      - "--disable-documentation"
+      - "--disable-seccomp"
+      - "--disable-sandboxed-triggers"
+      - "--disable-system-helper"
+      - "--with-system-install-dir=/var/lib/flatpak"
+    cleanup:
+      - "/etc/profile.d"
+      - "/lib/systemd"
+      - "/share/dbus-1/interfaces/org.freedesktop.*"
+      - "/share/dbus-1/services/org.freedesktop.*"
+      - "/share/gdm"
+    sources:
+      - type: archive
+        url: "https://github.com/flatpak/flatpak/releases/download/1.14.8/flatpak-1.14.8.tar.xz"
+        sha256: "1016b7327f7af87896f95465f7e5813750d3b7049a3740a1a4ddcb5fa8c5348e"
+        x-checker-data:
+          type: "json"
+          url: "https://api.github.com/repos/flatpak/flatpak/releases/latest"
+          version-query: ".tag_name"
+          url-query: "\"https://github.com/flatpak/flatpak/releases/download/\\($version)/flatpak-\\($version).tar.xz\""
+# Install flatrun
+  - name: just
```

Comment:
Is this build only dependency? If so you should add cleanup to remove it.

Reply:
Yes, everything in the /app directory up to build time is a build dependency up until the point the app gets installed.

I went ahead and removed just as a build dependency, as it was just running install commands instead. This cleans up the flatpak manifest a bit too.

Also, let me know if this other change is satisfactory, I now simply remove everything from the /app directory *after* building and *before* installing flatrun. This is because the BaseApp is there to ensure that libflatpak is available as a build dependency.